static/_redirects

```diff
@@ -169,6 +169,7 @@
 
 /docs/reference/deprecation-policy/     /docs/reference/using-api/deprecation-policy/ 301
 
+/docs/reference/generated/kubectl/kubectl_commands/     docs/reference/kubectl/ 301
```

Comment:
There are a couple of minor typos in the proposed fix.

```suggestion
/docs/reference/generated/kubectl/kubectl-commands/     /docs/reference/kubectl/ 301
```

Reply:
Added this fixes in new commit. Hope this will fix the build too.